conda-forge.yml

```diff
@@ -4,9 +4,11 @@ provider: {linux_aarch64: default, linux_ppc64le: native}
 test_on_native_only: true
 bot:
   abi_migration_branches:
-  - 3.7
-  - 3.8
-  - 3.9
+  - "3.7"
```

Comment:
We can remove 3.7, it's been dropped in conda-forge since last October (and EOL upstream since this July)